### ISSUE TITLE
Can't connect network.

### ISSUE THREAD

**Issue by strayedstar** [raspberrypi/linux#6565]
### Describe the bug

System:Rbpi OS
I build a kernel by the way:
![image](https://github.com/user-attachments/assets/307e88a3-7049-4fc9-b8cf-fae16b3d9ca8)
but after flash and boot,I can't get any network connect.
dmesg -k
![image](https://github.com/user-attachments/assets/f71ffcfe-0bb0-4046-a4c0-297c42b41568)
even set a static ip and route manually,it still can't work.

### Steps to reproduce the behaviour

build a kernel and boot it

### Device (s)

Raspberry Pi 5

### System

![image](https://github.com/user-attachments/assets/0a6b74f6-a1dc-4687-a552-4b9488539c90)
kernel version:6.6.67-rpi/6.13-rc3-rpi

**Comment by kaysievers** [comment]
I see the same. The wired Ethernet does not transmit packages.

**Comment by DrBest3787** [comment]
On Raspberry 4 the Ethernet AND the WiFi works. Dis you Check the cables and Adapters

**Comment by strayedstar** [comment]
> On Raspberry 4 the Ethernet AND the WiFi works. Dis you Check the cables and Adapters

Yes,I recheck it,but unhelpful,the adapters don't accept any package.
Mainline kernel already support Raspberry Pi 4,so you can do it easy.

**Comment by kaysievers** [comment]
> I see the same. The wired Ethernet does not transmit packages.

Updating the device tree files in /boot seems to fix it for me.
